### PR TITLE
Vid

### PR DESCRIPTION
**Pull request by izhuxiaoqing**

### CODE REVIEW

File: docs-2.0/1.introduction/3.vid.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+In Nebula Graph, a vertex is uniquely identified by its ID, which is called a VID, or a Vertex ID.
+
+## Features
+
+- The data types of VIDs are restricted to `FIXED_STRING(<N>)`, or `INT64`; a graph space can only select one VID type.
+
+- There is only one VID in a graph space. It functions just as a primary key in a relational database. VIDs in different graph spaces are utterly independent.
+
+- The VID generation method must be set by users, because Nebula Graph does not provide self-increasing ID, or UUID.
```
Comment (whitewum) [inline docs-2.0/1.introduction/3.vid.md:11]: self-increasing -> auto increasing
tech term

File: docs-2.0/1.introduction/3.vid.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## VID Operation
+
+- Nebula Graph 1.x only supports `INT64`, and Nebula Graph 2.x supports `INT64`, and `FIXED_STRING(<N>)`. In `CREATE SPACE`, VID types can be set via `vid_type`.
+
+- `id()` function can be used to set, or cite a VID.
+
+- `LOOKUP`, or `MATCH` statements can be used to find a VID via property index.
+
+- Direct access to vertices statements via VIDs enjoys peak performance, such as `DELETE xxx WHERE id(xxx) == "player100"`, or `GO FROM "player100"`. Finding VIDs via properties and then operating the graph will cause poor performance, such as `LOOKUP | GO FROM $-.ids`, which will run both `LOOKUP`, and `|` one more time.
```
Comment (randomJoe211) [inline docs-2.0/1.introduction/3.vid.md:31]:
```suggestion
- Direct access to vertices statements via VIDs enjoys peak performance, such as `DELETE xxx WHERE id(xxx) == "player100"` or `GO FROM "player100"`. Finding VIDs via properties and then operating the graph will cause poor performance, such as `LOOKUP | GO FROM $-.ids`, which will run both `LOOKUP`, and `|` one more time.
```

File: docs-2.0/1.introduction/3.vid.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## VID Generation
+
+VIDs can be generated via applications. Here are some tips:
+
+- (Optimal) Directly take a unique primary key or property as a VID; property access depends on the VID.
+
+- Generate a VID via a unique combination of properties; property access depends on property index.
+
+- Generate a VID via algorithms like snowflake; property access depends on property index.
```
Comment (randomJoe211) [inline docs-2.0/1.introduction/3.vid.md:41]: Using semicolons makes no sense here. What do you think?

File: docs-2.0/1.introduction/3.vid.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+- If short primary keys greatly outnumber long primary keys, do not enlarge the `N` of `FIXED_STRING(<N>)` too much. Otherwise, it will occupy a lot of memory and hard disks, and downgrade performance. Generate VIDs via BASE64, MD5, hash by encoding and splicing.
```
Comment (whitewum) [inline docs-2.0/1.introduction/3.vid.md:43]: downgrade performance -> slower performance?

**Review by randomJoe211 — APPROVED** [review]

File: docs-2.0/1.introduction/3.vid.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+- A VID in a graph space is unique. It functions just as a primary key in a relational database. VIDs in different graph spaces are independent.
+
+- The VID generation method must be set by users, because Nebula Graph does not provide auto increasing ID, or UUID.
+
+- Vertices with the same VID will be identified as the same one. For example:
+  
+  + A VID is the unique identifier of an entity, like a person's ID card number. A tag means the type of an entity,  such as driver, and boss. Different tags define two groups of different properties, such as driving license number, driving age, order amount, order taking alt, and job number, payroll, debt ceiling, business phone number.
+
+  + When two `INSERT` statements (neither uses a parameter of `IF NOT EXISTS`) with the same VID and tag are operated at the same time，the latter `INSERT` will cover the former.
```
Comment (whitewum) [inline docs-2.0/1.introduction/3.vid.md:17]: cover-> overwirte

File: docs-2.0/1.introduction/3.vid.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+- `LOOKUP` or `MATCH` statements can be used to find a VID via property index.
+
+- Direct access to vertices statements via VIDs enjoys peak performance, such as `DELETE xxx WHERE id(xxx) == "player100"` or `GO FROM "player100"`. Finding VIDs via properties and then operating the graph will cause poor performance, such as `LOOKUP | GO FROM $-.ids`, which will run both `LOOKUP` and `|` one more time.
+
+## VID Generation
+
+VIDs can be generated via applications. Here are some tips:
+
+- (Optimal) Directly take a unique primary key or property as a VID. property access depends on the VID.
```
Comment (whitewum) [inline docs-2.0/1.introduction/3.vid.md:37]: VID.  what is this dot ?

Comment (whitewum) [reply docs-2.0/1.introduction/3.vid.md:37]: Property ?

**Review by whitewum — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**